setup.py

```diff
@@ -92,6 +92,9 @@ def check_argument(compiler, arg):
         extra += ['-Werror=unknown-warning-option']
     if check_argument(compiler, '-Werror=unused-command-line-argument'):
         extra += ['-Werror=unused-command-line-argument']
+    # silence clang for unused gcc CFLAGS added by Debian
+    if check_argument(compiler, '-Wno-unused-command-line-argument'):
+        extra += ['-Wno-unused-command-line-argument']
```

Comment:
Doesn't this disable the unused-command-line-argument error which we need in this function?

Reply:
TBH I didn't look at what check_argument actually does, so quite possible I did something wrong here :)